cfme/tests/containers/test_pause_resume_workers.py

```diff
@@ -0,0 +1,76 @@
+from cfme.containers.provider import ContainersProvider
+from cfme.utils.appliance.implementations.ui import navigate_to
+import pytest
+
+
+pytestmark = [
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(1),
+    pytest.mark.provider([ContainersProvider], scope='function')
+]
+
+
+def get_pause_resume_buttons(view):
+    # ensure resume option is exists
+    resume_option = filter(lambda item: "resume" in item.lower(), view.toolbar.configuration.items)
+    assert resume_option, "Not resume provider option is available in the configuration menu"
+    resume_option = resume_option.pop()
+
+    # ensure pause option is exists
+    pause_option = filter(lambda item: "pause" in item.lower(), view.toolbar.configuration.items)
+    assert pause_option, "Not pause provider option is available in the configuration menu"
+    pause_option = pause_option.pop()
+
+    return pause_option, resume_option
+
+
+def check_ems_state_in_diagnostics(appliance, provider):
+    workers_view = navigate_to(appliance.collections.diagnostic_workers, 'AllDiagnosticWorkers')
+    workers_view.browser.refresh()
+    ems_worker_is_running = False
+    for row in workers_view.workers_table.rows():
+        name = getattr(row, "Name")
```

Comment:
`getattr()` raises exception, thus the assert below is moot since the exception will occur here, not set 'None'. A default must be passed to `getattr`.